tests/interactive_broker_version.py

```diff
@@ -52,6 +58,9 @@ def test_version (version, cmd=None, deploy=True, conf={}, debug=False, exec_cnt
 
     brokers = []
     for n in range(0, broker_cnt):
+        # Configure rack & replica selector if broker supports fetch-from-follower
+        if version_as_number(version) >= 2.4:
+            defconf.update({'conf': ['broker.rack=RACK${appid}', 'replica.selector.class=org.apache.kafka.common.replica.RackAwareReplicaSelector']})
```

Comment:
This needs to go in cluster_testing.py as well.

But, I'm not sure we want to change the default test environment to use fff for all tests since that changes our test coverage.
Instead I suggest you make this an option to interactive_broker_version.py and broker_version_test.py and add a new Make target to set up a rack-aware environment and run all tests, see the `broker_idempotence` target for an example.

Reply:
notes:
- all the other tests will have `client.rack` set to empty and `org.apache.kafka.common.replica.RackAwareReplicaSelector` will never result in a preferred replica being sent in a FETCH response in that case, given the configured `broker.rack` values, so in practice the changes in this PR won't change code coverage of the other tests.
- the fff test checks that messages were fetched from a replica (via rxbytes in stats), so it will fail if run in a non rack aware environment. This test would need to be aware of it's environment (I guess that is not hard, but I don't know).